cypress/integration/template-starter/main.js

```diff
@@ -4,5 +4,6 @@ describe('Main view', () => {
   it('should render page', () => {
     cy.login({ redirectToUri: URL_EXAMPLES_STARTER });
     cy.getByText('Hello, world').should('exist');
+    cy.percySnapshot();
```

Comment:
seems like this is the only snapshot uploaded...

Reply:
Hmm weird...

Reply:
It seems like they create 2 builds. Not sure if this is the intended behaviour

<img width="1354" alt="image" src="https://user-images.githubusercontent.com/1110551/62612723-4335ff80-b908-11e9-9e69-ad504048a012.png">

Reply:
Hm yeah, I don't think that's what we want 🤔

Reply:
It kind of makes sense though because we run cypress on 2 different apps...maybe we need to have 2 different projects

Reply:
I created 2 projects, it should work fine now